geoips/plugins/modules/readers/seviri_hrit.py

```diff
@@ -226,6 +229,7 @@ def get_top_level_metadata(fnames, sect):
     #     raise HritError('Unknown satname encountered: {}'.format(msg_satname))
     md["platform_name"] = msg_satname[0:3] + "-" + msg_satname[-1]  # msg-1 or msg-4
     md["source_name"] = "seviri"
+    md["source_file"] = source_file
```

Comment:
See xarray standards: https://github.com/NRLMMD-GEOIPS/geoips/blob/main/docs/source/devguide/xarray_standards.rst#xarray-standard-attributes

Reply:
Gotcha. I changed this to adhere to the xarray_standards you sent.

Reply:
[Here](https://github.com/NRLMMD-GEOIPS/geoips/pull/427/files#r1821413678) is where it was resolved.

Reply:
Thanks!